packages/smooth_app/lib/pages/preferences_v2/roots/dev_mode_root.dart

```diff
@@ -0,0 +1,666 @@
+                    .getFlag(userPreferencesFlagPricesReceiptMultiSelection) ??
+                false,
+            onToggle: (bool value) async {
+              await userPreferences.setFlag(
+                userPreferencesFlagPricesReceiptMultiSelection,
+                value,
+              );
+
+              if (!context.mounted) {
+                return;
+              }
+
+              _showSuccessMessage(context, appLocalizations);
+            },
+          ),
+          TogglePreferenceTile(
+            title: 'User ordered knowledge panels',
+            state: userPreferences.getFlag(userPreferencesFlagUserOrderedKP) ??
+                false,
+            onToggle: (bool value) async {
+              await userPreferences.setFlag(
+                  userPreferencesFlagUserOrderedKP, value);
+
+              if (!context.mounted) {
+                return;
+              }
+
+              _showSuccessMessage(context, appLocalizations);
+            },
+          ),
+          PreferenceTile(
+            title: 'Temporary access to location search',
+            onTap: () async {
+              final LocalDatabase localDatabase = context.read<LocalDatabase>();
+              final DaoOsmLocation daoOsmLocation =
+                  DaoOsmLocation(localDatabase);
+              final List<OsmLocation> osmLocations =
+                  await daoOsmLocation.getAll();
+              if (!context.mounted) {
+                return;
+              }
+              final List<SearchLocationPreloadedItem> preloadedList =
+                  <SearchLocationPreloadedItem>[];
+              for (final OsmLocation osmLocation in osmLocations) {
+                preloadedList.add(
+                  SearchLocationPreloadedItem(
+                    osmLocation,
+                    popFirst: false,
+                  ),
+                );
+              }
+              final OsmLocation? osmLocation =
+                  await Navigator.push<OsmLocation>(
+                context,
+                MaterialPageRoute<OsmLocation>(
+                  builder: (BuildContext context) => SearchPage(
+                    SearchLocationHelper(),
+                    preloadedList: preloadedList,
+                    autofocus: false,
+                  ),
+                ),
+              );
+              if (osmLocation == null) {
+                return;
+              }
+              await daoOsmLocation.put(osmLocation);
+              if (!context.mounted) {
+                return;
+              }
+              ScaffoldMessenger.of(context).showSnackBar(
+                SnackBar(
+                  content: Text(
+                    osmLocation.getTitle() ??
+                        osmLocation.getSubtitle() ??
+                        osmLocation.getLatLng().toString(),
+                  ),
+                ),
+              );
+            },
+          ),
+          PreferenceTile(
+            title: 'Preference Search...',
```

Comment:
Can we get rid of it, since we're making it mainstream ? @PrimaelQuemerais @monsieurtanuki

Reply:
@teolemon That's the search _within_ the preferences, like "do they have a 'dark mode' option in the preferences". It's already mainstream, isn't it?